canonical-websites/www.ubuntu.com
# Validate tempate

## Done
Validated the base template so we only see in page errors

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Go to http://0.0.0.0:8001/support/community-support
- Validate the markup and see it passes
- Check against live markup

## Issue / Card
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/1993
